pkg/istio-agent/sds-agent_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istioagent
+
+import (
+	"testing"
+)
+
+func TestSDSAgentWithEmptyCAProvider(t *testing.T) {
+	fm := fileMountedCertsEnv
+	fileMountedCertsEnv = true
+	defer func() { fileMountedCertsEnv = fm }()
+	// Validate that SDS server can start without any error.
```

Comment:
Can you add comments to explain what this function is testing? Is it testing that NewSDSAgent() does not return an error when CA_PROVIDER is empty? If so, why do we need to test CA_PROVIDER (since fileMountedCertsEnv is used to express the meaning of whether proxy uses file mounted certificates or not)?

Reply:
Testname is incorrect. Will change